tests/test_cli_units.py

```diff
@@ -79,3 +91,12 @@ def test_clear_virtual_links_no_devices(self, mock):
             self.assertEquals(res, [])
             self.assertEqual(ctx.output, ['WARNING:root:Cannot clear virtual links: no network interfaces provided.'])
         mock.assert_not_called()
+
+    def test_netplan_try_ready_stamp(self):
+        stamp_file = os.path.join(self.tmproot, 'run', 'netplan-try.ready')
+        cmd = NetplanTry()
+        self.assertFalse(os.path.isfile(stamp_file))
+        cmd.touch_ready_stamp()
+        self.assertTrue(os.path.isfile(stamp_file))
+        cmd.clear_ready_stamp()
```

Comment:
(nitpick) maybe a test that runs clear_ready_stamp() again when the file does not exist to check that it behaves correctly in this case (i.e. check the expected error or silently ignore error, whatever is the right thing here).